schemas/stsci.edu/asdf/wcs/celestial_frame-0.1.0.yaml

```diff
@@ -0,0 +1,30 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/wcs/celestial_frame-0.1.0"
+tag: "tag:stsci.edu:asdf/wcs/celestial_frame-0.1.0"
+
+title: >
+  Represents a celestial frame.
+
+allOf:
+  - type: object
+    properties:
+      reference_position:
+        description: |
+          The position of the reference frame.
+        enum: [geocenter, barycenter, heliocenter]
```

Comment:
Remove `reference_position` from `celestial_frame` and add it to `spectral_frame`.